packages/components/src/components/pagination/component.tsx

```diff
@@ -76,8 +76,8 @@ export class KolPagination implements PaginationAPI {
 				} else if (ellipsis === true) {
 					ellipsis = false;
 					return (
-						<li>
-							<span class="separator" key={`${this.nonce}-el-${page}`} aria-hidden="true"></span>
+						<li key={nonce()}>
```

Comment:
Ich bin nicht ganz zufrieden mit dieser Lösung, habe aber keinen funktionierenden Ansatz mit deterministischen Keys (wie in der V1) gefunden.

Vielleicht ergeben die unique keys aber auch Sinn, weil sich beim Navigieren in der Paginierung tatsächlich die Elemente, aber auch die Eigenschaften der Elemente, bei jedem Render ändern?

In jedem Fall bin ich hier offen für andere Lösungen und Ideen.

Reply:
Ich halte das für korrekt